driver/include/sensor.h

```diff
@@ -82,6 +85,7 @@ typedef enum {
     FRAMESIZE_96X96,    // 96x96
     FRAMESIZE_QQVGA,    // 160x120
     FRAMESIZE_QCIF,     // 176x144
+    FRAMESIZE_200X200,  // 200x200
```

Comment:
can we have this new frame size in a separate PR? Why is it needed?

Reply:
A separate [request](https://github.com/espressif/esp32-camera/pull/456) has been made, PTAL, Thanks.  
Only some projects must use this resolution, or do we have other methods to add a custom resolution? Some projects tend to use a custom resolution.

Reply:
There are some "hacky" ways, but maybe we need to think of adding an option to do it. Adding to the list as-is could/will break some projects that expect that list to be static. Another option would be to have a call to get a list of available resolutions.

Reply:
Maybe there will be some custom resolution requirements later, so if possible, please add a method to support this custom resolution.